src/components/sideContent/sideContent.module.scss

```diff
@@ -0,0 +1,25 @@
+.headerContent {
+    display: flex;
+    font-size: var(--fontsize-body-m);
+    margin-top: var(--spacing-2-xs);
+    
+    svg {
+      margin-right: var(--spacing-xs);
+    }
+
+    a svg {
+      vertical-align: bottom;
+      margin-left: var(--spacing-xs);
+      margin-right: 0;
+    }
+  }
+
```

Comment:
We should add a bit of margin between the subregion elements. Tapahtumat.hel.fi has `margin-bottom: var(--spacing-l);` on each wrapper div.